### PR TITLE
Adding Height Converter Project

### PR DESCRIPTION
**Pull request by Drishti-jain21**
Closes issue #14 

I am adding height converter project made using HTML, CSS, JavaScript.

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ Once your PR gets merged, then don't forget to add your Dev Info in [Contributor
 |     7.     | Music library   | [Music library](https://music-library-one.vercel.app/)           | [Music library](https://github.com/Ashish-khanagwal/Awesome-web-dev/tree/main/Dhruvi's-Frontend-Project/MUSIC) |
 |     8.     | Hotel website   | [Hotel website](https://template-dusky-rho.vercel.app/)          | [Hotel website](https://github.com/Ashish-khanagwal/Awesome-web-dev/tree/main/Dhruvi's-Frontend-Project/DAY%20(TEMPLATE)) |
 |     9.     | Swiggy home page | [Swiggy home page](https://50-projects-50-days-gnjg.vercel.app/) | [Swiggy home page](https://github.com/Ashish-khanagwal/Awesome-web-dev/tree/main/Dhruvi's-Frontend-Project/DAY%20(SWIGGY'S%20HOME%20PAGE)) |
+|     10.     | Swiggy home page | [Height Converter](https://636524da483a130a61a91992--heightconverterweb.netlify.app/) | [Height Converter](https://github.com/Drishti-jain21/Awesome-web-dev/tree/main/Dhruvi's-Frontend-Project/Drishti(Height%20Converter)) |
```
Comment (Ashish-khanagwal) [inline README.md:56]: Please make sure to enter your project's name here

**Review by Ashish-khanagwal — CHANGES_REQUESTED** [review]

**Comment by Ashish-khanagwal** [comment]
@Drishti-jain21 Don't add your project to Dhruvi's projects, add them separately as other projects added

**Comment by Drishti-jain21** [comment]
I have changed
please check

**Comment by Ashish-khanagwal** [comment]
please change the code base link in README.md file too, navigating to 404 page not found

**Comment by Drishti-jain21** [comment]
done
please check

**Review by Ashish-khanagwal — APPROVED** [review]
Code looks good to me

Thank you for your valuable contribution

**Comment by Ashish-khanagwal** [comment]
@Drishti-jain21 Please add your Dev Info in [CONTRIBUTORS.md](https://github.com/Ashish-khanagwal/Awesome-web-dev/blob/main/CONTRIBUTORS.md) file

**Pull request merged**